### PR TITLE
Add Dragos Platform modeling rule

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-8434)

## Description
Add modeling rule for Dragos Platform logs.

### CODE REVIEW

File: Packs/Dragos_Platform/pack_metadata.json
Comment (OBavly) [inline Packs/Dragos_Platform/pack_metadata.json:1]: Looks good - Let's just add keywords and tags

**Review by OBavly — CHANGES_REQUESTED** [review]

File: Packs/Dragos_Platform/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<~XSIAM>
+
+### This pack includes:
+- Log Normalization - XDM mapping for key event types.
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:4]:
```suggestion
Log Normalization - XDM mapping for key event types.
```

File: Packs/Dragos_Platform/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- Alerts
+- Events
+
+***
+
+## Data Collection
+To configure Dragos Platform to send logs to Cortex XSIAM, follow the below steps.
+
+### Dragos Platform side
+1. Logon to Dragos as an administrator.
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:16]:
```suggestion
1. Log in to Dragos as an administrator.
```

File: Packs/Dragos_Platform/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+2. Navigate to **Admin** &rarr; **Syslog** or browse to "https://<your-site-store>/syslog/".
+3. Go to the **SERVERS** tab and click on **+ ADD SERVER**.
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:18]:
```suggestion
3. Go to the **SERVERS** tab and click **+ ADD SERVER**.
```

File: Packs/Dragos_Platform/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+4. Fill the below data:
+
+    | Parameter                             | Value                                                                                 |
+    |:--------------------------------------|:--------------------------------------------------------------------------------------|
+    | Name                                  | Cortex XSIAM Broker VM.                                                               |
+    | Hostname/IP                           | Enter the Broker VM IP Address.                                                       |
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:24]:
```suggestion
    | Hostname/IP                           | Enter the Broker VM IP address.                                                       |
```

File: Packs/Dragos_Platform/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    | Port                                  | Enter the syslog service port that you want to use for sending logs to the Broker VM. |
+    | Protocol                              | TCP/TLS.                                                                              |
+    | Source Hostname                       | Leave the default value / set a value of your choice.                                 |
+    | Source Process                        | Leave the default value / set a value of your choice.                                 |
+    | TLS Protocol Configuration (optional) | If protocol set to TLS, set all the relevant values.                                  |
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:29]:
```suggestion
    | TLS Protocol Configuration (optional) | If protocol is set to TLS, set all the relevant values.                                  |
```

File: Packs/Dragos_Platform/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+5. Check the "RFC 5424 Modern Syslog" checkbox under **Message Format**.
+6. Check the "Use newline delimiter for TCP and TLS streams" checkbox under **Message Delimiter**.
+7. Click on **Next: SET TEMPLATE**.
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:33]:
```suggestion
7. Click **Next: SET TEMPLATE**.
```

File: Packs/Dragos_Platform/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    | Name                                  | Cortex XSIAM Broker VM.                                                               |
+    | Hostname/IP                           | Enter the Broker VM IP Address.                                                       |
+    | Port                                  | Enter the syslog service port that you want to use for sending logs to the Broker VM. |
+    | Protocol                              | TCP/TLS.                                                                              |
+    | Source Hostname                       | Leave the default value / set a value of your choice.                                 |
+    | Source Process                        | Leave the default value / set a value of your choice.                                 |
+    | TLS Protocol Configuration (optional) | If protocol set to TLS, set all the relevant values.                                  |
+
+5. Check the "RFC 5424 Modern Syslog" checkbox under **Message Format**.
+6. Check the "Use newline delimiter for TCP and TLS streams" checkbox under **Message Delimiter**.
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:32]:
```suggestion
5. Check the **RFC 5424 Modern Syslog** checkbox under **Message Format**.
6. Check the **Use newline delimiter for TCP and TLS streams** checkbox under **Message Delimiter**.
```

File: Packs/Dragos_Platform/README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+Follow the below steps to configure the Broker VM to receive Dragos Platform logs.
+
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**.
+2. Go to the **APPS** column under the **Brokers** tab and add the **Syslog** app for the relevant broker instance. If the **Syslog** app already exists, hover over it and click **Configure**.
+3. Click **Add New**.
+4. When configuring the Syslog Collector, set the following parameters:
+
+    | Parameter    | Value                                                                                                                                           |
+    |:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------|                  
+    | `Protocol`   | Select **UDP** for the default forwarding, **TCP** or **Secure TCP** (depends on the protocol you configured in Dragos Platform configuration). |
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:54]:
```suggestion
    | `Protocol`   | Select **UDP** for the default forwarding, **TCP** or **Secure TCP** (depends on the protocol you configured in the Dragos Platform configuration). | 
```

File: Packs/Dragos_Platform/README.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**.
+2. Go to the **APPS** column under the **Brokers** tab and add the **Syslog** app for the relevant broker instance. If the **Syslog** app already exists, hover over it and click **Configure**.
+3. Click **Add New**.
+4. When configuring the Syslog Collector, set the following parameters:
+
+    | Parameter    | Value                                                                                                                                           |
+    |:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------|                  
+    | `Protocol`   | Select **UDP** for the default forwarding, **TCP** or **Secure TCP** (depends on the protocol you configured in Dragos Platform configuration). | 
+    | `Port`       | Enter the syslog service port that Cortex XSIAM Broker VM should listen on for receiving forwarded events from Dragos Platform.                 |
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:55]:
```suggestion
    | `Port`       | Enter the syslog service port that Cortex XSIAM Broker VM should listen on for receiving forwarded events from the Dragos Platform.                 |
```

File: Packs/Dragos_Platform/README.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    | `Vendor`     | Enter dragos.                                                                                                                                   |
+    | `Product`    | Enter platform.                                                                                                                                 |
```
Comment (ShirleyDenkberg) [inline Packs/Dragos_Platform/README.md:57]:
```suggestion
    | `Vendor`     | Enter **dragos**.                                                                                                                                   |
    | `Product`    | Enter **platform**.                                                                                                                                 |
```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

File: Packs/Dragos_Platform/pack_metadata.json
Comment (yasta5) [reply Packs/Dragos_Platform/pack_metadata.json:1]: Added

**Review by OBavly — APPROVED** [review]

**Pull request merged**